examples/gmail/honcho_gmail.py

```
@@ -0,0 +1,355 @@
+        for m in msgs:
+            register_peer(m["from"])
+            for addr in parse_address_list(m["to"]):
+                register_peer(addr)
+            for addr in parse_address_list(m["cc"]):
+                register_peer(addr)
+            for addr in parse_address_list(m["bcc"]):
+                register_peer(addr)
+
+    # Summary
+    total_msgs = sum(len(v) for v in all_thread_messages.values())
+    print(f"\nSummary:")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)